dxr/plugins/descriptor/__init__.py

```diff
@@ -45,8 +49,8 @@ class FileToIndex(dxr.indexers.FileToIndex):
     """Do lots of work to yield a description needle."""
 
     comment_re = re.compile(r'(?:.*?/\*+)(?:\s*\*?\s*)(?P<description>.*?)(?:(?:\*+/.*)|(?:$))', flags=re.M)
-    docstring_re1 = re.compile(r'"""\s*(?P<description>[^"]*)', flags=re.M)
-    docstring_re2 = re.compile(r"'''\s*(?P<description>[^']*)", flags=re.M)
+    docstring_res = [re.compile(r'"""\s*(?P<description>[^"]*)', flags=re.M),
+                     re.compile(r"'''\s*(?P<description>[^']*)", flags=re.M)]
     title_re = re.compile(r'<title>([^<]*)</title>')
```

Comment:
I guess the title tag could have attrs, but we can come and fix that if it ever matters.